packages/material-ui-lab/src/Alert/Alert.d.ts

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { StandardProps } from '@material-ui/core';
+import { PaperProps } from '@material-ui/core/Paper';
+
+export type Color = 'success' | 'info' | 'warning' | 'error';
+
+export interface AlertProps extends StandardProps<PaperProps, AlertClassKey, 'variant'> {
+  /**
+   * The action to display.
+   */
+  action?: React.ReactNode;
+  /**
+   * Override the default text for the *close popup* icon button.
+   *
+   * For localization purposes, you can use the provided [translations](/guides/localization/).
+   */
+  closeText?: string;
+  /**
+   * Main color for the Alert, picked from theme palette.
+   */
+  color?: Color;
+  /**
+   * The icon element placed before the children.
+   */
+  icon?: React.ReactNode | false;
+  /**
+   * The role attribute of the element.
```

Comment:
This description doesn't really say enough.  I'd suggest something to add but I don't, myself, really understand what it does.  It's an accessibility thing or for the snackbar?

Reply:
It's somewhat related to the Snackbar. I propose we link this page: `/components/alert/#accessibility`.